asab/library/providers/zookeeper.py

```diff
@@ -111,6 +111,10 @@ def __init__(self, library, path):
 		if z_url is None and url_pieces.netloc == "" and url_pieces.path == "" and self.Zookeeper.Path != '':
 			self.BasePath = '/' + self.Zookeeper.Path
 
+		# Handle `zk://./path` configuration
+		if z_url is None and url_pieces.netloc == "." and self.Zookeeper.Path != '':
```

Comment:
`url_pieces.netloc.startswith(".")` ?

Reply:
It doesn't starts with `'.'`  

`url_pieces = ParseResult(scheme='zk', netloc='.', path='/path' ...`